docs/endpoints/cookieSync.md

```diff
@@ -0,0 +1,37 @@
+# Handling cookie syncs.
+
+## POST /cookie_sync
+
+This returns a set of URLs to enable cookie syncs across bidders. (See Prebid.js documentation?) The request
+must supply a JSON object to define the list of bidders that may need to be synced.
+
+```
+{
+    "uuid": "blahblah",
+    "bidders": ["appnexus", "rubicon"]
+}
+```
+
+The `uuid` field is due to be deprecated, as it is no longer needed. It just gets passed through unused into the response currently.
```

Comment:
If the endpoint works without it, don't document it here. With no documentation, it's unlikely anyone will start using it. If it's mentioned, then some people will. Also: it complicates the docs & wastes time for anyone who just wants to know how to use it properly.

Honestly... Kendall said prebid.js wasn't using it, and Prebid Mobile doesn't use Cookies. If we're going to make breaking changes, sooner is better than later. I think we should just delete it from the endpoint now.

Reply:
It is now deprecated completely. :) Bandaid yanked!